src/awkward/forms/form.py

```diff
@@ -422,4 +422,19 @@ def length_zero_array(
             backend=backend,
             highlevel=highlevel,
             behavior=behavior,
+            simplify=False,
+        )
+
+    def length_one_array(self, *, backend=numpy_backend, highlevel=True, behavior=None):
+        return ak.operations.ak_from_buffers._impl(
+            form=self,
+            length=1,
+            container={
+                "": b"\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00"
+            },
+            buffer_key="",
+            backend=backend,
+            highlevel=highlevel,
+            behavior=behavior,
+            simplify=False,
```

Comment:
You're right! That's true; it needs to be at least as large as the largest primitive itemsize, and it also needs to be at least as large as two int64s for an `offsets` array of length 2 (for a ListOffsetArray of length 1). That's what I was thinking of here.

In awkward/types/numpytype.py, we have the complete list of supported dtypes, and the code only provisionally adds some types because NumPy doesn't have them on all platforms. The above would support any 128-bit types, do we have any 256-bit types? (`complex256`?)

Reply:
Yes, it's okay for that `bytes` object to be _longer_ than it needs to be. It needs to be at least 256 bits (32 bytes) so that complex256 doesn't break.